.travis.yml

```diff
@@ -20,7 +20,7 @@ before_install:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 
-install: ./gradlew assemble --stacktrace
+install: ./gradlew :applicationUi:assemble --stacktrace
```

Comment:
This should be "ui", not "applicationUi" to match everthing else (such as the package name)